gatsby/src/docs/clients/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+draft: false
+categories: []
+toc: true
+title: Legacy Clients
+sidebar_order: 4
+tags: []
+---
+Some of the clients that report to Sentry have been replaced by newer SDKs or are no longer maintained.  We document them here for users who are still using those clients but they are recommended against for new projects.
+
+<div class="container mt-4 mb-3">
```

Comment:
Is this the expected output?

Reply:
its not awesome, but we probably never have to update this page again, so i think it will be ok (ill visually test the page tho)

Reply:
fyi same concern on /platforms/, but we can also just come back and change this later really easily. it renders as we'd expect (and want).